bites_test.py

```diff
@@ -296,32 +321,40 @@ def test_one_whole_round_is_played(self, render_game_mock):
       "purple": None}
     anthill_after_turn_2_luigi = starting_anthill
     anthill_food_tokens_after_turn_2_luigi = starting_anthill_food_tokens
-    fake_luigi = mock.MagicMock()
-    fake_luigi.take_turn = mock.MagicMock(return_value = (
-      trail_after_turn_2_luigi, ant_pos_after_turn_2_luigi, anthill_after_turn_2_luigi, anthill_food_tokens_after_turn_2_luigi))
-    
-    ants = ["red", "yellow", "green", "brown", "purple"]
-    tokens_for_trail = {}
-    players = [fake_mario, fake_luigi]
-    bites_game = Bites(ants, tokens_for_trail, players)
     
     expected_new_trail = trail_after_turn_2_luigi
     expected_new_ant_positions = ant_pos_after_turn_2_luigi
     expected_new_anthill = starting_anthill
     expected_new_anthill_food_tokens = starting_anthill_food_tokens
-
+    
+    fake_mario = mock.MagicMock()
+    fake_luigi = mock.MagicMock()
+    players = [fake_mario, fake_luigi]
+    ants = ["red", "yellow", "green", "brown", "purple"]
+    tokens_for_trail = {}
+    anthill_order = ""
+    bites_game = Bites(ants, tokens_for_trail, players, anthill_order)
+    
     bites_game.trail = starting_trail
     bites_game.ant_positions = starting_ant_positions
     bites_game.anthill = starting_anthill
+    bites_game.anthill_order = [4, 3, 2, 1, 0]
+    
+    fake_mario.take_turn = mock.MagicMock(side_effect = [(
+      trail_after_turn_1_mario, ant_pos_after_turn_1_mario, anthill_after_turn_1_mario, bites_game.anthill_order, anthill_food_tokens_after_turn_1_mario),
```

Comment:
Take turn is not returning the anthill order, therefore the mock shouldn't either.